va/va.go

```diff
@@ -524,3 +542,26 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 
 	return body, url.String(), nil
 }
+func alpnaddr(addr string) (arpa string, err error) {
+	const hexDigit = "0123456789abcdef"
```

Comment:
You should be able to use `fmt` and an `%x` parameter to avoid needing to do this hex conversion yourself.